client/src/Movies/MovieList.js

```diff
@@ -24,7 +26,16 @@ export default class MovieList extends Component {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <Link to={`/movies/${movie.id}`} className='link'>
+            <MovieCard
+              key = {movie.id}
+              movie = {movie}
+              title = {movie.title}
+              director = {movie.director}
+              metascore = {movie.metascore}
+              stars = {movie.stars}
```

Comment:
again, don't have to send down the individual properties if you send the entire object